core/server/src/main/java/alluxio/master/file/meta/InodeTree.java

```diff
@@ -864,6 +873,12 @@ private void addInodeFromJournalInternal(Inode<?> inode) {
     }
     parentDirectory.addChild(inode);
     mInodes.add(inode);
+    // If journal entry has no security enabled, change the replayed inode permission to be 0777
+    // for backwards-compatibility.
+    if (SecurityUtils.isSecurityEnabled() && inode != null && inode.getOwner().isEmpty()
```

Comment:
This one perhaps not possible, unless `InodeFileEntry` can represent a null `InodeFile`